ouroboros-consensus/src/Ouroboros/Consensus/NodeNetwork.hs

```diff
@@ -261,42 +263,86 @@ protocolCodecs cfg version = ProtocolCodecs {
           (nodeEncodeHeaderHash (Proxy @blk))
           (nodeDecodeHeaderHash (Proxy @blk))
 
-    , pcBlockFetchCodecSerialised =
-        codecBlockFetchSerialised
-          (nodeEncodeHeaderHash (Proxy @blk))
-          (nodeDecodeHeaderHash (Proxy @blk))
+    , pcBlockFetchCodecSerialised = staticBlockFetchCodecSerialised
 
-    , pcTxSubmissionCodec =
-        codecTxSubmission
-          nodeEncodeGenTxId
-          nodeDecodeGenTxId
-          nodeEncodeGenTx
-          nodeDecodeGenTx
+    , pcTxSubmissionCodec         = staticTxSubmissionCodec
 
-    , pcLocalChainSyncCodec =
-        codecChainSyncSerialised
-          (encodePoint (nodeEncodeHeaderHash (Proxy @blk)))
-          (decodePoint (nodeDecodeHeaderHash (Proxy @blk)))
-          (encodeTip   (nodeEncodeHeaderHash (Proxy @blk)))
-          (decodeTip   (nodeDecodeHeaderHash (Proxy @blk)))
+    , pcLocalChainSyncCodec       = staticLocalChainSyncCodec
 
-    , pcLocalTxSubmissionCodec =
-        codecLocalTxSubmission
-          nodeEncodeGenTx
-          nodeDecodeGenTx
-          (nodeEncodeApplyTxError (Proxy @blk))
-          (nodeDecodeApplyTxError (Proxy @blk))
+    , pcLocalTxSubmissionCodec    = staticLocalTxSubmissionCodec
 
-    , pcLocalStateQueryCodec =
-        codecLocalStateQuery
-          (encodePoint (nodeEncodeHeaderHash (Proxy @blk)))
-          (decodePoint (nodeDecodeHeaderHash (Proxy @blk)))
-          nodeEncodeQuery
-          nodeDecodeQuery
-          nodeEncodeResult
-          nodeDecodeResult
+    , pcLocalStateQueryCodec      = staticLocalStateQueryCodec
     }
 
+data NodeToClientCodecs blk failure m bytesLCS bytesLTX bytesLSQ = NodeToClientCodecs {
+    nodeToClientChainSyncCodec       :: Codec (ChainSync (Serialised blk) (Tip blk))
+                                         failure m bytesLCS
+  , nodeToClientTxSubmissionCodec    :: Codec (LocalTxSubmission (GenTx blk) (ApplyTxErr blk))
+                                         failure m bytesLTX
+  , nodeToClientStateQueryCodec      :: Codec (LocalStateQuery blk (Query blk))
+                                         failure m bytesLSQ
```

Comment:
Are these `::` aligned with those in `ProtocolCodecs`? That's not necessary, just align these three with each other, like `stylish-haskell` does.